inputfiles/overridingTypes.json

```diff
@@ -267,18 +333,42 @@
         "signatures": ["createCaption(): HTMLTableCaptionElement"]
     },
     {
+        "kind": "property",
+        "interface": "HTMLTableElement",
+        "name": "rows",
+        "type": "HTMLCollectionOf<HTMLTableRowElement>"
+    },
+    {
+        "kind": "property",
+        "interface": "HTMLTableElement",
+        "name": "tBodies",
+        "type": "HTMLCollectionOf<HTMLTableSectionElement>"
+    },
+    {
         "kind": "method",
         "interface": "HTMLTableSectionElement",
         "name": "insertRow",
         "signatures": ["insertRow(index?: number): HTMLTableRowElement"]
     },
     {
+        "kind": "property",
+        "interface": "HTMLTableSectionElement",
+        "name": "rows",
+        "type": "HTMLCollectionOf<HTMLTableRowElement>"
+    },
+    {
         "kind": "method",
         "interface": "HTMLTableRowElement",
         "name": "insertCell",
         "signatures": ["insertCell(index?: number): HTMLTableCellElement"]
     },
     {
+        "kind": "property",
+        "interface": "HTMLTableRowElement",
+        "name": "cells",
+        "type": "HTMLCollectionOf<HTMLTableCellElement>"
```

Comment:
as per https://html.spec.whatwg.org/multipage/tables.html#dom-tr-cells, this should be `HTMLTableCellElement | HTMLTableHeaderCellElement`